base/letter.dtx

```diff
@@ -178,7 +178,7 @@
 % \maketitle
 % \tableofcontents
 %
-% \StopEventually
+% \MaybeStop
```

Comment:
No `{}`?

Reply:
so true :-) but it was always missing ... it works because it picks up the following `\par`. Added a brace group